crates/nu-command/tests/commands/assignment/append_assign.rs

```diff
@@ -1,80 +1,93 @@
 use nu_test_support::nu;
 
+fn assert_true(output: String) {
+    print!("{}", output);
```

Comment:
I don't think this extra `print!` is required.

Reply:
It was there before, so I left it in.  Removed it and the function, since it is no longer needed.